std_msgs/Header header
std_msgs/Float32MultiArray coordinates
rail_object_detector/Object[] yolo_objects
std_msgs/Bool mutual
std_msgs/Float32 mutual_value
std_msgs/String nearest_object
std_msgs/Int32 frame_count